pointcloud/src/test/scala/geotrellis/pointcloud/raster/ept/IDWRasterSourceSpec.scala

```diff
@@ -35,15 +35,15 @@ class IDWRasterSourceSpec extends FunSpec with RasterMatchers {
         name        = "src/test/resources/red-rocks/",
         crs         = CRS.fromEpsgCode(26913),
         cellType    = DoubleCellType,
-        gridExtent  = new GridExtent(Extent(481968.0, 4390186.0, 482856.0, 4391074.0), 6.9375, 6.9375, 128, 128),
-        resolutions = CellSize(6.9375, 6.9375) :: Nil,
+        gridExtent  = new GridExtent(Extent(481968.0, 4390186.0, 482856.0, 4391074.0), 0.216796875, 0.216796875, 4096, 4096),
+        resolutions = List(CellSize(6.9375,6.9375), CellSize(3.46875,3.46875), CellSize(1.734375,1.734375), CellSize(0.8671875,0.8671875), CellSize(0.43359375,0.43359375), CellSize(0.216796875,0.216796875)),
```

Comment:
Ah, it looks like is hould be reversed, to have a similar behavior to the rest of RasterSources.

Reply:
An example of how `GDALRasterSource` and `GeoTiffRasterSource` behave

```scala
import geotrellis.raster.geotiff._
import geotrellis.raster.gdal._

GDALRasterSource("multiband_co.tif").resolutions
// List(
//   CellSize(500.00000000000136,500.00000000000136),
//   CellSize(1000.0000000000035,1000.0000000000035), 
//   CellSize(2000.000000000007,2000.000000000007), 
//   CellSize(4000.000000000014,3984.848484848499), 
//   CellSize(8000.000000000028,7969.696969696998), 
//   CellSize(16000.000000000056,15939.393939393996)
// )

GeoTiffRasterSource("multiband_co.tif").resolutions
//> List(
//   CellSize(500.00000000000176,500.00000000000176), 
//   CellSize(1000.0000000000035,1000.0000000000035), 
//   CellSize(2000.000000000007,2000.000000000007), 
//   CellSize(4000.000000000014,3984.848484848499), 
//   CellSize(8000.000000000028,7969.696969696998),
//   CellSize(16000.000000000056,15939.393939393996)
// )
```

Reply:
This should be properly documented in [`RasterMetadata`](https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/RasterMetadata.scala) if the order of resolutions is important.  There's no mention there, so I didn't order things in any particular way.

Reply:
@jpolchlo ah yeah; there was a fresh issue about making that consistent in the gt repo a week ago: https://github.com/locationtech/geotrellis/issues/3210 and we forgot to add the mention of ordering into string docs during the review thanks